src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -38,6 +38,9 @@ class ApiProjectKey:
 class ApiOrganization:
```

Comment:
I'm starting to prefer renaming this to `ThinOrganization`?  Any thoughts?
In general, trying to find a nice consistent naming schema that translates to "a serialized, minimal version of an original ORM, for the sake of region <-> control" hahaha.

Reply:
Names are hard, we already have 'detailed', 'light-weight' and 'summary' forms of organizations. Some other options that come to mind are:

* OrganizationEntity
* OrganizationSketch
* RemoteOrganization
* OrganizationMimic

There are probably other suffixes/prefixes that could work. Regardless of which one we end up choosing using that convention consistently is the important part.